src/lib/deterministic/fetchers/client.ts

```
@@ -187,7 +187,7 @@ export class GitHubClient {
     const url = path.startsWith("http") ? path : `https://api.github.com${path}`;
     const requestHeaders = new Headers(options.headers);
     const acceptHeader = requestHeaders.get("accept") ?? "application/json";
-    const cacheKey = `rest:${this.tokenFingerprint}:${acceptHeader}:${url}`;
+    let cacheKey = `rest:${this.tokenFingerprint}:${acceptHeader}:${url}`;
```

ESLint

[error] 'cacheKey' is never reassigned. Use 'const' instead.

(prefer-const)